package.json

```diff
@@ -82,8 +83,7 @@
   "lint-staged": {
     "*.ts": [
       "yarn prettier --write",
-      "eslint --fix",
-      "bash .github/workflows/scripts/kebab-case.sh"
```

Comment:
We should remove this if it's still in our template!

Reply:
An oversight, my mistake

Reply:
To clarify I meant that the `ts-template` should be updated.

Reply:
@Keyrxng are you looking into this or should I look into this?

Reply:
marking it as resolved since the kebab case line is not present here or in ts-template anymore.

Reply:
@EresDev I'm sorry I didn't notice you had tagged me here my apologises I'm unsure how it slipped passed me